src/test/run-pass/issue-48508-aux.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Shouldn't this file be in the auxiliary folder?

Reply:
Everything in `auxillary` is treated as a crate. This needs to be a module to trigger #48508

Reply:
Ah, ok